lib/process-services-cloud/src/lib/process/start-process/components/start-process-cloud.component.ts

```diff
@@ -171,6 +176,21 @@ export class StartProcessCloudComponent implements OnChanges, OnInit, OnDestroy
         return !!this.processDefinitionCurrent?.formKey;
     }
 
+    get defaultStartProcessButtonLabel(): string {
+        return this.translateService.instant('ADF_CLOUD_PROCESS_LIST.ADF_CLOUD_START_PROCESS.FORM.ACTION.START').toUpperCase();
+    }
+
+    get defaultCancelProcessButtonLabel(): string {
+        return this.translateService.instant('ADF_CLOUD_PROCESS_LIST.ADF_CLOUD_START_PROCESS.FORM.ACTION.CANCEL').toUpperCase();
+    }
+
+    constructor(private translateService: TranslationService) {
+        this.startProcessButtonLabel = this.translateService
+            .instant('ADF_CLOUD_PROCESS_LIST.ADF_CLOUD_START_PROCESS.FORM.ACTION.START')
+            .toUpperCase();
+        this.cancelButtonLabel = this.translateService.instant('ADF_CLOUD_PROCESS_LIST.ADF_CLOUD_START_PROCESS.FORM.ACTION.CANCEL').toUpperCase();
+    }
```

Comment:
```suggestion
    constructor(private translateService: TranslationService) {
        this.startProcessButtonLabel = this.defaultStartProcessButtonLabel;
        this.cancelButtonLabel = this.defaultCancelProcessButtonLabel;
    }
```

Reply:
yep I did the refactoring and forgot to apply here :(